### PR TITLE
Add an exponent to the accuracy portion of score

### PR DESCRIPTION
**Pull request by smoogipoo**
Awards higher accuracy values exponentially more score. This is the same curve as ScoreV2.

### CODE REVIEW

File: osu.Game/Rulesets/Scoring/ScoreProcessor.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class ScoreProcessor : JudgementProcessor
     {
         private const double base_portion = 0.3;
```
Comment (peppy) [inline osu.Game/Rulesets/Scoring/ScoreProcessor.cs:19]: think we should rename this to `accuracy_portion`? not sure if it works better than "base"..

**Review by peppy — DISMISSED** [review]

File: osu.Game/Rulesets/Scoring/ScoreProcessor.cs (hunk 1 above)
Comment (smoogipoo) [reply osu.Game/Rulesets/Scoring/ScoreProcessor.cs:19]: I was going to do that in the upcoming mania PR but can do it here if you'd rather that.

Comment (peppy) [reply osu.Game/Rulesets/Scoring/ScoreProcessor.cs:19]: Either or.

**Comment by smoogipoo** [comment]
Let's not do this for now because it feels bad, and change mania instead.

**Pull request closed without merging**